gov2/bedrock-runtime/hello/hello.go

```diff
@@ -0,0 +1,85 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// snippet-start:[gov2.bedrock-runtime.Hello]
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/bedrockruntime"
+)
+
+// Each model provider defines their own individual request and response formats.
+// For the format, ranges, and default values for the different models, refer to:
+// https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters.html
+
+type ClaudeRequest struct {
+	Prompt            string   `json:"prompt"`
+	MaxTokensToSample int      `json:"max_tokens_to_sample"`
+	// Omitting optional request parameters
+}
+
+type ClaudeResponse struct {
+	Completion string `json:"completion"`
+}
+
+// main uses the AWS SDK for Go (v2) to create an Amazon Bedrock Runtime client 
+// and invokes Anthropic Claude 2 inside your account and the chosen region.
+// This example uses the default settings specified in your shared credentials
+// and config files.
+func main() {
+
+	region := "us-east-1"
+	sdkConfig, err := config.LoadDefaultConfig(context.Background(), config.WithRegion(region))
```

Comment:
Either don't specify region (i.e., take whatever is configured in .aws/config) or make it an input. I'm in `us-west-2` and it took me a while to discover why I kept getting 403 errors.

Reply:
See my [response above](https://github.com/awsdocs/aws-doc-sdk-examples/pull/5679#discussion_r1406465793)

Reply:
See my [response above](https://github.com/awsdocs/aws-doc-sdk-examples/pull/5679#discussion_r1407970951)